lib/bing_ads_ruby_sdk/services/base.rb

```diff
@@ -18,7 +18,12 @@ def call(operation_name, message = {})
           camelized_name,
           preprocess(message, camelized_name)
         )
-        postprocess(response)
+        processed_response = postprocess(response)
+        BingAdsRubySdk::Errors::ErrorHandler.new(processed_response).call
```

Comment:
OK so we needed to go through `processed_response` before checking for errors because it includes the `snakize` call, right?

Reply:
absolutely